app/frontend/tests/unit/components/designer/FormViewer.spec.js

```diff
@@ -294,7 +294,7 @@ describe('FormViewer.vue', () => {
   });
 
   it('onMounted', () => {
```

Comment:
I think the way this is usually done is that the `describe` elements are on the outside, with a collection of `it` elements on the inside. That way you can have multiple nested levels of `describes`.